cmd/execute.go

```diff
@@ -109,9 +112,18 @@ func executeTransformers() {
 	}
 
 	if len(ethStorageInitializers) > 0 {
-		storageHealthCheckMessage := []byte("storage watcher starting\n")
+		storageHealthCheckMessage := []byte("storage watcher for new diffs starting\n")
+		statusWriter := fs.NewStatusWriter(healthCheckFile, storageHealthCheckMessage)
+		sw := watcher.NewStorageWatcher(&db, newDiffBlockFromHeadOfChain, statusWriter, watcher.New)
+		sw.AddTransformers(ethStorageInitializers)
+		wg.Add(1)
+		go watchEthStorage(&sw, &wg)
+	}
+
+	if len(ethStorageInitializers) > 0 {
```

Comment:
pretty minor but we could probably combine these operations if they have the same predicate, right?